manifest.json

```
@@ -2,18 +2,19 @@
     "components": [
         {
             "id": "devcontainer-setup",
-            "type": "setup",
+            "description": "Basic setup of the devContainer for VApp development.",
+            "basePath": "setup",
```

could we also add src here? or is the basepath not build for such usage?
```suggestion
            "basePath": "setup/src",
```